---
title: 'Phytobiome Metabarcoding: A Tool to Help Identify Prokaryotic and Eukaryotic Causal Agents of Undiagnosed Tree Diseases'
authors:
  - Carrie J. Fearer
  - antonino
  - Cristina Rosa
  - Pierluigi Bonello

#author_notes:
# - 'Equal contribution'
# - 'Equal contribution'
date: '2022-07-13T00:00:00Z'
doi: '10.1007/978-1-0716-2517-0_19'

# Schedule page publish date (NOT publication's date).
# publishDate: '2017-01-01T00:00:00Z'

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ['6']

# Publication name and optional abbreviated publication name.
#publication: ' Plant Pathology Method and Protocols'
#publication_short: ''

abstract: Recent advancements in high-throughput sequencing have provided scientists with vastly enhanced tools to diagnose unknown tree diseases. One of these techniques is referred to as metabarcoding, which uses phylogenetically informative reference genes to taxonomically classify short DNA sequences amplified from environmental samples. Using metabarcoding, we are able to compare the microbiota of symptomatic and asymptomatic (including presumably na√Øve) samples and identify microbe(s) that are only present in symptomatic samples and could therefore be responsible for the undiagnosed disease. Metabarcoding involves two main steps: library preparation and bioinformatic processing. For library preparation, the appropriate reference gene for the organism of interest (i.e., bacteria, phytoplasma, fungi, or other eukaryotes, such as nematodes) is amplified from the DNA extracted from the environmental samples using PCR and prepared for sequencing. The bioinformatic processing includes four major steps: (1) quality check and cleanup on raw reads; (2) classification of the sequences into taxonomically informative groups (ASVs or OTUs); (3) taxonomy assignments based on the reference database; and (4) differential abundance and diversity analyses to identify microbes that are significantly associated with just symptomatic samples and that point toward the putative causal agent of the disease.

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
  - Source Themes
featured: false

# links:
# - name: ""
#   url: ""
# url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
#image:
#  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/jdD8gXaTZsc)'
#  focal_point: ''
#  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: [microbiome, methods]

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides:
---
